src/blend/src/Shared/Blend/Blend.lua

```diff
@@ -548,6 +575,48 @@ function Blend.Children(parent, value)
 	end
 end
 
+--[=[
+	Mounts attributes to the parent and returns an object which will cleanup and delete
+	all attributes when removed.
+
+	```lua
+	local attribute = Blend.State("abc")
+	Blend.New "ScreenGui" {
+		Parent = game.Players.LocalPlayer.PlayerGui;
+		[Blend.Attributes] = {
+			Attribute1 = attribute;
+			Attribute2 = 123;
+		};
+	};
+	```
+	Cleanup:
+	* Attributes will be cleaned up on unsubscribe
+
+	@param parent Instance
+	@param value any
+	@return MaidTask
+]=]
+function Blend.Attributes(inst, value)
+	assert(typeof(inst) == "Instance", "Bad instance")
+	return Blend.ComputedPairs(value, function(attributeName, attributeValue, innerMaid)
+		local function getValue(v)
+			if type(v) == "table" then
+				if ValueObject.isValueObject(v) then
+					return getValue(v.Value)
+				elseif Brio.isBrio(v) then
+					return getValue(v:GetValue())
```

Comment:
nit: Should probably keep lifetime of attribute application to the brio lifetime. I think you can do 

```lua
innerMaid[attributeName] = v:GetDiedSignal():Connect(function()
    inst:SetAttribute(attributeName, attributeValue)
end)
```

You'll also need to set
innerMaid[attributeName] = nil

Before you do each item.


This should only occur if we emit a brio with another brio in it, so we could also remove the `if Brio.isBrio` condition entirely. Up to you.

Reply:
I'm still getting my sea legs with brio, so I left the if Brio.isBrio for now, but you're welcome to change it. I believe I've ported the rest of the code though!

Reply:
Sounds good.